Получить все рекомендации пользователя

Leilorr opened the pull request:
Условия задачи
Реализовать возможность получения всех рекомендаций, которые были даны одному пользователю.

Leilorr committed:
Получить все рекомендации пользователя
Условия задачи
Реализовать возможность получения всех рекомендаций, которые были даны одному пользователю.

vladmeniailov reviewed (approved):
На мой взгляд всё в порядке.

InnMish commented on src/main/java/school/faang/user_service/dto/recommendation/RecommendationDto.java:
```diff
@@ -0,0 +1,12 @@
+@AllArgsConstructor
+@NoArgsConstructor
+public class RecommendationDto {
+    private long receiverId;
```
Когда пользователь отправляет рекомендацию, он же заполняет и другие поля тоже 
Посмотри в entity -> Recommendation, какие еще полезные поля тут нужно написать 
Чтобы пользователь, который дает рекомендацию, мог заполнить больше информации

Leilorr committed:
Получить все рекомендации пользователя
Условия задачи
Реализовать возможность получения всех рекомендаций, которые были даны одному пользователю.
